<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>USNavigationMarkerPlacement</class>
 <widget class="QWidget" name="USNavigationMarkerPlacement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QmitkUSNavigationProcessWidget" name="navigationProcessWidget" native="true"/>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startExperimentButton">
       <property name="text">
        <string>Start New Experiment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="finishExperimentButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Finish Experiment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="runningLabel">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkUSNavigationProcessWidget</class>
   <extends>QWidget</extends>
   <header>src/internal/Widgets/QmitkUSNavigationProcessWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>OnStartExperiment()</slot>
  <slot>OnFinishExperiment()</slot>
 </slots>
</ui>
